main.js

```diff
@@ -302,15 +302,17 @@ const helpMenu = [{
     label: 'Request a Feature',
     sublabel: 'Create a new feature request on GitHub',
     click: () => {
-        shell.openExternal(walcinfo.bugs.url + "/new/?template=feature_request.md&labels=enhancement&title=[Feature+Request]");
+        shell.openExternal('https://github.com/cstayyab/WALC/discussions/categories/feature-requests');
     }
-}, {
-    label: 'Vote for a Feature',
-    sublabel: 'Vote for existing features on FeatHub',
-    click: () => {
-        shell.openExternal("https://feathub.com/cstayyab/WALC");
-    }
-}, {
+}, 
+// {
```

Comment:
Instead of commenting the code, please use the `visible` property of `MenuItem` ([Find documentation here](https://www.electronjs.org/docs/api/menu-item#new-menuitemoptions))